internal/consensus/common_test.go

```diff
@@ -154,6 +157,10 @@ func signVote(vs *validatorStub, voteType types.SignedMsgType, chainID string, b
 	return v
 }
 
+func signVote(vs *validatorStub, voteType types.SignedMsgType, chainID string, blockID types.BlockID, extEnabled bool) *types.Vote {
```

Comment:
I don't understand why this is necessary if it's just a wrapper to avoid calling `vs.clock.Now()`...maybe I'm overlooking something?

Reply:
This is necessary because there are tens of calls to this method with the original format. All the methods in this section are essentially this: call another method with some defaults.